app/views/pages/feedbackpage.html.erb

```diff
@@ -0,0 +1,63 @@
+<div class="govuk-grid-row">
```

Comment:
There are a few things in this that don't match the screenshot in the ticket:
- No Home link
- No "satisfaction survey" title
- text sizes are different

This should also be using the form builder methods - https://github.com/x-govuk/govuk-form-builder.

Reply:
Yes, we always want to use the form builder methods. Please update to use them in the new page.